From dcd44c265ae50bce605ef3e1d5b7ad03644eb9ce Mon Sep 17 00:00:00 2001
From: Gerd Hoffmann <kraxel@redhat.com>
Date: Thu, 23 Jun 2011 12:40:49 -0300
Subject: [RHEL6 qemu-kvm PATCH 016/115] enable usb-storage + scsi-bus + scsi-disk

RH-Author: Gerd Hoffmann <kraxel@redhat.com>
Message-id: <1308832951-8995-14-git-send-email-kraxel@redhat.com>
Patchwork-id: 27858
O-Subject: [RHEL-6.2 kvm PATCH 013/115] enable usb-storage + scsi-bus + scsi-disk
Bugzilla: 561414 632299 645351 711354
RH-Acked-by: Hans de Goede <hdegoede@redhat.com>
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Kevin Wolf <kwolf@redhat.com>

Enable usb-storage and the scsi bits needed for it.

Signed-off-by: Gerd Hoffmann <kraxel@redhat.com>
---
 Makefile |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>
---
 Makefile |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/Makefile b/Makefile
index a023624..63fda6f 100644
--- a/Makefile
+++ b/Makefile
@@ -179,6 +179,7 @@ obj-y += eeprom93xx.o
 obj-y += cdrom.o
 obj-y += usb.o usb-hub.o usb-$(HOST_USB).o usb-hid.o usb-wacom.o
 obj-y += usb-serial.o usb-net.o usb-bus.o usb-desc.o
+obj-y += usb-msd.o scsi-bus.o scsi-disk.o
 obj-$(CONFIG_SSI) += ssi.o
 obj-$(CONFIG_SSI_SD) += ssi-sd.o
 obj-$(CONFIG_SD) += sd.o
-- 
1.7.3.2

